rllib/algorithms/algorithm_config.py

```diff
@@ -278,6 +280,18 @@ def __init__(self, algo_class=None):
             "intra_op_parallelism_threads": 8,
             "inter_op_parallelism_threads": 8,
         }
+        # Torch compile settings
+        self.torch_compile_learner_forward_train = False
```

Comment:
@kouroshHakha Actually, the compile() call itself takes almost no time. So it's safe to always compile both, inference and exploration, even if only one will be used.

Reply:
So I'll make it so that torch_compile_worker will result in compiling forward exploration and inference no matter what. I see no reason why this should hurt.